index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css"><link rel="stylesheet" 
+      href="https://use.fontawesome.com/releases/v5.15.4/css/all.css"
+      integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm"
+      crossorigin="anonymous">
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700&display=swap" rel="stylesheet">
+    <title>Document</title>
+</head>
+<body>
+    <header class="header">
+
+        <h3 class="logo__title">My Logo</h3>
+
+        <i class="fas fa-bars"></i> 
+        
+    </header>
+  
+    <main class="container">
+        <div class="intro">
+            <h1 class="intro__heading">
+                I’m Eloi
+              Glad to see you!
+        
+            </h1>
+            <p class="intro__text">
+                I’m a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! If you like what you 
+                see and have a project you need coded, don’t hestiate to contact me.</p>
+            <a href="#" class="intro__contact">let's connect</a>
+            <div class="intro__social--cont">
+                
+                <i class="fab fa-twitter"></i>
+                <i class="fab fa-linkedin-in"></i>
+                <i class="fab fa-medium-m"></i>
+                <i class="fab fa-github-square"></i>
+                <i class="far fa-hand-peace"></i>
+
+            </div>
```

Comment:
- [ ] It would be good to use a ul tag to group the icons instead of a div, an li per icon, and each icon should have a link(a tag). Please kindly fix that.